internal/clientconn/conn.go

```diff
@@ -228,7 +235,7 @@ func (c *conn) run(ctx context.Context) (err error) {
 		}
 
 		if resHeader == nil || resBody == nil {
-			c.l.Info("no response to send to client")
+			c.l.Debugf("no response to send to client")
```

Comment:
I'm not sure about this one, but the message itself doesn't look very informative.

Reply:
Is it true that If the header or body is nil there is no response sent to the client?

Reply:
@seeforschauer it seems so, yes. It looks a bit strange to me - there is not an error here and nothing else...

Reply:
the condition and the logging seem like an error - I expect that it was not sent only if both `resHeader` and `resBody` are nil.

Reply:
@AlekSi what do you think about this part of the code? It's not really straightforward which log level should be here, especially, if the client doesn't get any response... In which case it's possible that the response doesn't have any content? Is it a sign that something went wrong and we should panic? Or at least log an error?

Reply:
Fixed. As agreed in call, throw panic instead.